includes/class-wc-stripe-account.php

```diff
@@ -0,0 +1,160 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC_Stripe_Account class.
+ *
+ * Communicates with Stripe API.
+ */
+class WC_Stripe_Account {
+
+	const ACCOUNT_OPTION = 'wcstripe_account_data';
+	const ACCOUNT_API    = 'account';
+
+	public function __construct() {
+		self::cache_account();
```

Comment:
Would we want to try reading the cached data first before doing the API request?

Reply:
That's a valid concern. :+1:  
I will rearrange the logic or if I just call the `get_cached_account_data` here instead of `cache_account()` that should also work. Because all the checks are there sequentially.

Reply:
Yep, calling `get_cached_account_data()` looks right to me.